src/mod/endpoints/mod_sofia/sofia.c

```diff
@@ -7496,6 +7496,10 @@ static void sofia_handle_sip_i_state(switch_core_session_t *session, int status,
 		if (r_sdp) {
 			switch_channel_set_variable(channel, SWITCH_R_SDP_VARIABLE, r_sdp);
 
+			if(ss_state == nua_callstate_ready && status == 200){
```

Comment:
indentation matters. eg: space between "if" and "(" .
https://developer.signalwire.com/freeswitch/FreeSWITCH-Explained/Community/Contributing-Code/Coding-Guidelines_6586862/

Reply:
@dragos-oancea Thanks for review, have addressed review comment